app/models/outgoing_sms_communication.rb

```diff
@@ -0,0 +1,28 @@
+class OutgoingSmsCommunication < ApplicationRecord
+  belongs_to :user
+
+  enum message_type: [
+    :initial_message,
+    :cancellation_message,
+    :reminder_message
+  ]
+
+  def self.record(recip_email, status, type)
+    user = User.find_by(email: recip_email)
+    msg_type = type_map[type]
```

Comment:
Per our messing around with this live, instead of defining the `type_map` method, Rails makes available a method which is the plural of the enum that returns a hash which you can use to look up the digit by the string:
```
message_types[type]
```

Reply:
Done.